cmd/coder/main.go

```diff
@@ -15,31 +15,35 @@ import (
 	"go.coder.com/flog"
 )
 
-var (
-	version string = "unknown"
-)
+// Using a global for the version so it can be set at build time using ldflags.
+var version string = "unknown"
 
 func main() {
 	ctx, cancel := context.WithCancel(context.Background())
 	defer cancel()
 
+	// If requested, spin up the pprof webserver.
 	if os.Getenv("PPROF") != "" {
 		go func() {
 			log.Println(http.ListenAndServe("localhost:6060", nil))
 		}()
 	}
 
+	// TODO: Document why this is needed. Only does something on windows.
```

Comment:
I'd be inclined to avoid adding TODOs to the code base? I think it's better to either do the thing if its small or open issues?

Reply:
Will remove the todos in favor of issues.